pymagicc/core.py

```diff
@@ -149,24 +154,111 @@ def create_copy(self):
 
     @property
     def binary_name(self):
+        """
+        Name of the MAGICC binary file
+
+        Returns
+        -------
+        str
+            Name of the binary file
+        """
         return basename(self.executable)
 
     @property
     def original_dir(self):
+        """
+        Directory of the MAGICC package.
+
+        This is the directory which contains the ``run`` and ``out`` folders.
+
+        Returns
+        -------
+        str
+            Path of the MAGICC package
+        """
         return dirname(self.executable)
 
     @property
     def run_dir(self):
+        """
+        Run directory of the MAGICC package.
+
+        This path always ends in ``run``.
+
+        Returns
+        -------
+        str
+            Path of the run directory
+        """
         if self.root_dir is None:
             return None
         return join(self.root_dir, "run")
 
     @property
     def out_dir(self):
+        """
+        Output directory of the MAGICC package.
+
+        This path always ends in ``out``.
+
+        Returns
+        -------
+        str
+            Path of the output directory
+        """
         if self.root_dir is None:
             return None
         return join(self.root_dir, "out")
 
+    @property
+    def default_config(self):
+        """
+        Default configuration for a run
+
+        Returns
+        -------
+        :obj:`f90nml.Namelist`
+            Namelist object containing the default configuration
+        """
+        base = f90nml.read(join(self.run_dir, "MAGCFG_DEFAULTALL.CFG"))
+        user = f90nml.read(join(self.run_dir, "MAGCFG_USER.CFG"))
+        self._default_config = deepcopy(base)
+
+        def _deep_update(b, o):
+            for k, v in o.items():
+                if isinstance(v, dict):
+                    _deep_update(b[k], v)
+                else:
+                    b.update(o)
+
+        _deep_update(self._default_config, user)
+
+        return self._default_config
+
+    @property
+    def fgas_files_conc(self):
+        """
+        Flag used to set the F-gas concentration input files.
+
+        Returns
+        -------
+        str
+            Name of the flag
+        """
+        return "fgas_files_conc"
+
+    @property
+    def mhalo_switchfromconc2emis_year(self):
```

Comment:
Same here ... why is this a separate property?

Reply:
As above